### ISSUE TITLE
[ Feature Request ] Download specific version

### ISSUE THREAD

**Issue by Catwaii** [UndertakerBen/PorEdgeUpd#31]
A button for installing a custom version would be useful. I'd like to downgrade to 109 portable again

**Comment by ChenXiaoming233** [comment]
Same request, and this feature will be useful while we facing problems of 120+ versions not work.

**Comment by garry-ut99** [comment]
I agree, I need Edge 109 because it's the last version which runs in Win 7.
